### PR TITLE
Feature/Added tvOS-UITests

### PR DESCRIPTION
**Pull request by FraDeliro**
**Purpose**

This is a feature to add UITests for tvOS.

**A brief description of what has changed and why it has changed**

Added:

* Example_tvOSUITests.swift

**Ways to test feature or scenarios to try**

Run the tests for tvOS example app.

### CODE REVIEW

**Comment by bcylin** [comment]
Thanks for the pull request! Currently CI doesn't run Example-tvOSUITests. Can you also change `bundle exec rake "build:tvos[Example-tvOS]"` to `test:tvos[Example-tvOS]` in `.travis.yml`?

**Comment by bcylin** [comment]
Can you fix the colon violations and rebase the latest `develop` branch?

> Colon Violation: Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. (colon)

File: Example-tvOSUITests/ExampleUITests.swift [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  private lazy var app: XCUIApplication = XCUIApplication()
+
+  override func setUp() {
+    super.setUp()
+    continueAfterFailure = false
+    app.launch()
+  }
+
+  func testInteractions() {
```
[the remark this answers is not in the document]
Comment (FraDeliro) [reply Example-tvOSUITests/ExampleUITests.swift]: No worries ;) 👍

**Comment by bcylin** [comment]
Thanks @FraDeliro!

**Comment by FraDeliro** [comment]
Thank you @bcylin ;)

**Pull request merged**